generator/java/build.gradle

```diff
@@ -64,27 +69,26 @@ task generateClient(type: GradleBuild) {
             generateModelTests = false
             configOptions = [
                     groupId                  : 'com.criteo',
-                    apiPackage               : "${packageBase}.api".toString(), // com.criteo.api.marketingsolutions.v2021_10.api
-                    artifactId               : "criteo-api-${criteoService}-sdk-${technologyStack}".toString(), // criteo-api-marketingsolutions-sdk-java8
-                    artifactVersion          : "${packageVersion}".toString(), // v2021_10.1.211110 
-                    apiVersion               : parsedJson.info.version,  // 2021-10
-                    artifactDescription      : "Java SDK for Criteo API ${criteoService} for ${criteoApiVersion} version".toString(),
-                    artifactUrl              : 'https://github.com/criteo/criteo-api-java-sdk',
-                    modelPackage             : "${packageBase}.model".toString(),
-                    scmConnection            : 'scm:git:git://github.com/criteo/criteo-api-java-sdk.git',
-                    scmDeveloperConnection   : 'scm:git:ssh://github.com:criteo/criteo-api-java-sdk.git',
-                    scmUrl                   : 'https://github.com/criteo/criteo-api-java-sdk',
+                    apiPackage               : "${namespaceBase}.api".toString(), // com.criteo.api.marketingsolutions.v2021_10.api
+                    artifactDescription      : "${language.toUpperCase()} SDK for Criteo API ${criteoService} for ${criteoApiVersion} version".toString(),
+                    artifactId               : "criteo-api-${criteoService}-${versionCategory}-sdk".toString(), // criteo-api-marketingsolutions-preview-sdk
```

Comment:
I think we decided that the versionCategory should not be included in the package name right?

Reply:
We will update the story and take these latest decision into account in next commit